source/tutorial/acceptance-test.md

```diff
@@ -6,7 +6,7 @@ Here's a look at what we want to build before we get started.
 
 Let's work through what we want to do on the home page. We want our application to:
 
-* Show rentals on the home page
```

Comment:
@toddjordan, what is the difference between this and “List the available rentals” (line 12)?

In my opinion, it is either a repetition (as I assumed in https://github.com/emberjs/guides/pull/2025) or should say “Show rentals as the home page” to avoid repetition.

What do you think?